fdbrpc/FlowTransport.actor.cpp

```diff
@@ -122,7 +122,7 @@ struct EndpointNotFoundReceiver : NetworkMessageReceiver {
 	EndpointNotFoundReceiver(EndpointMap& endpoints) {
 		//endpoints[WLTOKEN_ENDPOINT_NOT_FOUND] = this;
 		Endpoint::Token e = WLTOKEN_ENDPOINT_NOT_FOUND;
-		endpoints.insert(this, e, TaskDefaultEndpoint);
+		endpoints.insert(this, e, static_cast<uint32_t>(TaskPriority::DefaultEndpoint));
```

Comment:
I had been scared off by the `uint32_t` instead of `int`, but looking deeper, it was just done for the bitmath that's done internally within EndpointMap, and if I'm doing a static_cast to uint32_t in all the call sites, I might as well just do it internally and let the API be clean.  Thanks!